wikichange/src/content/content.js

```diff
@@ -430,6 +433,36 @@ const highlightRevisionBetweenRevisionIds = async (title, curRevisionId, oldRevi
     }
 };
 
+/**
+ * Highlight taking advantage of the page tagging
+ * Right now I don't even use contex, because highlights are suppose to be in order
+ * @param {array of dictionary} context_array 
+ * @param {string} color 
+ */
+const highlightByMatchingMarks = async (context_array, color) => {
```

Comment:
Note that this will probably be somewhat bad if the highlighted word is something common like "the". it would be (I'm guessing) the first it sees. if this is just for the purpose of testing, i think it is fine. If this is being used, i don't think it is a good idea to be highlighting something that is not supposed to be highlighted (worse than not having any highlight at all)

Reply:
maybe adding a minimum constraints on how long the highlight has to be to use this method might be an okay work-around